apps/qubit/modules/digitalobject/actions/metadataComponent.class.php

```diff
@@ -351,4 +359,12 @@ protected function setPreservationCopyShowProperties()
     );
   }
 
+  protected function localizeUTCDatetime($dateTime)
```

Comment:
Nitpick: I think the method name should be `localizeUTCDateTime` (capital on "Time") to match the `$dateTime` variable capitalization, and the built-in [DateTime Class](https://www.php.net/manual/en/class.datetime.php)